src/CommunityToolkit.Maui/HandlerImplementation/Popup/Popup.windows.cs

```diff
@@ -8,12 +8,12 @@ public partial class Popup
 {
 	void OnPopupHandlerChanged(object? sender, EventArgs e)
 	{
-		if (Handler is null || Handler.NativeView is null)
+		if (Handler is null || Handler.PlatformView is null)
```

Comment:
We can use the same technique that you implemented elsewhere in this PR to shrink down this `if` statement 

```suggestion
		if (Handler?.PlatformView is null)
```